### ISSUE TITLE
Test rules

### ISSUE THREAD

**Issue by hmuniz** [own-pt/cl-conllu#22]
In [4dca2b](https://github.com/own-pt/cl-conllu/commit/4dca2bd80734df85ef30b94b226892cd78b10a68) I solved the merge conflict and tested the corte-e-costura function. 

The test was done using this [conllu file](https://github.com/own-pt/bosque-UD/blob/master/documents/CF104.conllu) and these rules:

```
(=> ((?a (= upostag "ADV") (= lemma "além"))
     *
     (?b (= lemma "ter")))
    ((?b (+ lemma "============"))))

(=> (?a (= lemma "liberdade") (= upostag "NOUN"))
    (?a (! upostag "===========")))

(=> ((?a (= lemma "trabalho"))
     ?
     (?b (= lemma "passar"))
     ?
     (?c (= lemma "em")))
    ((?a (! lemma "============"))))
```

**Comment by arademaker** [comment]
Não entendi o operador `?`, o `*` quer dizer "um ou mais tokens" mas o `?` ? Ele refere-se à opcionalidade do token anterior ou posterior? A notação não está boa.

**Comment by GPPassos** [comment]
I'm sorry, but what is the issue here? Is this just a report on what was done?

**Comment by arademaker** [comment]
Yes, @GPPassos , @hmuniz should have had created the issue before the code and report the evolution here, but since he didn't do it, I asked him to open the issue  anyway. Github does not send alerts for commits, only for updates in the issues.

**Comment by GPPassos** [comment]
Oh, no problem! It'd just be good to have a brief description of the problem as well in order to have a record

**Comment by arademaker** [comment]
I agree

**Comment by GPPassos** [comment]
@hmuniz Sorry if it's a silly question, but on https://github.com/own-pt/cl-conllu/blob/4dca2bd80734df85ef30b94b226892cd78b10a68/rules.lisp#L9 what do `rls` and `rhs` stand for?
I'd think that `rhs` means "right-hand side", but then instead of `rls` shouldn't it be `lhs` (left-hand side)?

**Comment by arademaker** [comment]
the corte-costura based rules were implemented, but I believe #50 can be more useful in the future. Anyway, it seems this issue was already solved.

**Comment by arademaker** [comment]
implemented by @Kalysthos closing it.
